bitbots_bringup/scripts/check_robot.py

```diff
@@ -79,12 +88,34 @@ def check_pressure(msg, min, max, foot_name):
         okay = False
     return okay
 
+got_pwm = False
+def pwm_callback(msg:JointState):
+    global got_pwm
+    for effort in msg.effort:
+        if effort = 100:
+            print_warn("Servo reported max PWM value. It is working at its limit!\n")
+
+#todo set correctly
```

Comment:
Todos should be promoted to github issues. This one also seems rather important and should probably be addressed before merging, since the current values would cause the script to throw incorrect warnings.